### ISSUE TITLE
Support upgrades of the instance

### ISSUE THREAD

**Issue by pavolloffay** [grafana/tempo-operator#296]
```[tasklist]
### Tasks
- [x] Create (empty for now)  upgrade procedure
- [x] Write upgrade test that simulates upgrade on OCP (using OLM and bundle)
```

**Comment by andreasgerstmayr** [comment]
@pavolloffay how should the upgrade test work?

Something like
* install the currently released 0.1.0 from operatorhub
* build a bundle from the current git commit
* upgrade via OLM and make sure the images in the deployments are replaced?

**Comment by pavolloffay** [comment]
👍🏼  LGTM
